core/spec/models/spree/variant_spec.rb

```diff
@@ -812,11 +812,27 @@
   end
 
   describe '#gallery' do
-    let(:product) { Spree::Variant.new }
-    subject { product.gallery }
+    let(:variant) { build_stubbed(:variant) }
+    subject { variant.gallery }
 
     it 'responds to #images' do
       expect(subject).to respond_to(:images)
     end
+
+    context "when variant.images is empty array" do
```

Comment:
Technically it should be an `ActiveRecord::Relation` not an array 🤓. I'd avoid specifying what it is at all here, `... is empty` is enough IMO.

Reply:
Agreed. Done